app/assets/javascripts/webauthn.js

```diff
@@ -124,10 +124,10 @@
           })
         })
       }).then(function (response) {
-        handleResponse($SUBMIT, $ERROR, response)
-      }).catch(function (error) {
-        setError($SUBMIT, $ERROR, error)
-      })
+        return response.text();
+      }).then((html) => {
+        document.body.innerHTML = html
+      });
```

Comment:
Currently, the handleResponse function actually does the redirect (by setting the window location) so without that the page will not know where to redirect to in the UI after sign in 
<img width="845" alt="image" src="https://user-images.githubusercontent.com/8496209/205101364-69a4d7b5-52db-421e-9cd3-995a3654515e.png">